File: CS/LayoutViewCustomButtonContent/LayoutViewCustomButtonSkinPaintStyle.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using DevExpress.XtraGrid.Registrator;
using DevExpress.XtraGrid.Views.Base;
using DevExpress.XtraGrid.Views.Layout.Drawing;

namespace CustomButtonsLayoutView
{
    class LayoutViewCustomButtonSkinPaintStyle : DevExpress.XtraGrid.Registrator.LayoutViewSkinPaintStyle
    {
        public override DevExpress.XtraGrid.Views.Base.BaseViewPainter CreatePainter(BaseView view)
        {
            return new LayoutViewCustomButtonPainter(view);
        }
    }
}

File: VB/LayoutViewCustomButtonContent/LayoutViewCustomButtonSkinPaintStyle.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text
Imports DevExpress.XtraGrid.Registrator
Imports DevExpress.XtraGrid.Views.Base
Imports DevExpress.XtraGrid.Views.Layout.Drawing

Namespace CustomButtonsLayoutView
	Friend Class LayoutViewCustomButtonSkinPaintStyle
		Inherits DevExpress.XtraGrid.Registrator.LayoutViewSkinPaintStyle
		Public Overrides Function CreatePainter(ByVal view As BaseView) As DevExpress.XtraGrid.Views.Base.BaseViewPainter
			Return New LayoutViewCustomButtonPainter(view)
		End Function
	End Class
End Namespace
